diff --git a/build/standalone.gypi b/build/standalone.gypi
index dad05ae..a350984 100644
--- a/build/standalone.gypi
+++ b/build/standalone.gypi
@@ -47,7 +47,7 @@
                   s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/mips.*/mips/")',
             }, {
               # OS!="linux" and OS!="freebsd" and OS!="openbsd" and OS!="netbsd"
-              'host_arch%': 'ia32',
+              'host_arch%': 'x64',
             }],
           ],
         },
